src/app/wallet/wallet.component.ts

```diff
@@ -83,39 +80,19 @@ export class WalletComponent implements OnInit {
     )
   }
 
-  manageWallets() {
+  manageVenlyWallets(): Observable<unknown> {
     return this.venly.manageWallets().pipe(
       concatMap(res => res ? this.signerService.login(this.venly, {force: false}) : EMPTY),
     )
   }
-
-  copyAddressToClipboard() {
-    this.sessionQuery.address$.pipe(take(1))
-      .subscribe(address => {
-        const selBox = document.createElement('textarea')
-        selBox.style.position = 'fixed'
-        selBox.style.left = '0'
-        selBox.style.top = '0'
-        selBox.style.opacity = '0'
-        selBox.value = address || ''
-        document.body.appendChild(selBox)
-        selBox.focus()
-        selBox.select()
-        document.execCommand('copy')
-        document.body.removeChild(selBox)
-        
-        this.copyLabelSub.next("Copied")
-        setTimeout(() => { this.copyLabelSub.next("Copy") }, 2000)
-      })
-  }
 }
 
-export enum TransactionType {
-  Investment = "Investment",
-  DividendPayout = "Dividend Payout"
+enum TransactionType {
+  Investment = 'Investment',
+  DividendPayout = 'Dividend Payout'
 }
 
-export interface WalletTransaction {
```

Comment:
Don't export interfaces by default if they are not used outside its file (or component's template).